### PR TITLE
[LIB-151] add get_user_info method to Connection

### PR DESCRIPTION
**Pull request by opalczynski**

### CODE REVIEW

File: syncano/connection.py [hunk 1]
```diff
@@ -380,12 +381,29 @@ def authenticate_user(self, **kwargs):
         return self.user_key
 
     def get_account_info(self, api_key=None):
-        if api_key:
-            self.api_key = api_key
+        self.api_key = api_key if api_key else self.api_key
```
Comment (Zhebr) [inline syncano/connection.py]: could be
```self.api_key = api_key or self.api_key```

File: syncano/connection.py [hunk 2]
```diff
@@ -380,12 +381,29 @@ def authenticate_user(self, **kwargs):
...
         if not self.api_key:
             raise SyncanoValueError('api_key is required.')
+
         return self.make_request('GET', self.ACCOUNT_SUFFIX, headers={'X-API-KEY': self.api_key})
 
+    def get_user_info(self, api_key=None, user_key=None):
+        self.api_key = api_key if api_key else self.api_key
+        self.user_key = user_key if user_key else self.user_key
+
+        if not self.api_key:
```
Comment (Zhebr) [inline syncano/connection.py]: wonder if this could be DRIED

File: syncano/connection.py [hunk 3]
```diff
@@ -380,12 +381,29 @@ def authenticate_user(self, **kwargs):
...
+        self.api_key = api_key if api_key else self.api_key
+
         if not self.api_key:
             raise SyncanoValueError('api_key is required.')
+
         return self.make_request('GET', self.ACCOUNT_SUFFIX, headers={'X-API-KEY': self.api_key})
 
+    def get_user_info(self, api_key=None, user_key=None):
+        self.api_key = api_key if api_key else self.api_key
+        self.user_key = user_key if user_key else self.user_key
```
Comment (opalczynski) [inline syncano/connection.py]: :dancer:

File: syncano/connection.py (hunk 2 above)
Comment (opalczynski) [inline syncano/connection.py]: Could be!

**Comment by Zhebr** [comment]
neat

**Pull request merged**